src/Versity.Users/External/Presentation/Program.cs

```diff
@@ -0,0 +1,33 @@
+using Domain.Models;
+using Infrastructure.Persistence;
+using Microsoft.AspNetCore.Identity;
+using Presentation.Extensions;
+
+var builder = WebApplication
+    .CreateBuilder(args)
+    .ConfigureBuilder();
+
+var app = builder.Build();
+app.ConfigureApplication();
+
+using var scope = app.Services.CreateScope();
+var serviceProvider = scope.ServiceProvider;
+try
```

Comment:
What is it?!

Reply:
This is changed in the following commit -> [Handle Exceptions, that are not handled by the middleware](https://github.com/jejikeh/versity/pull/3/commits/2fa194c8903248bb5f9d7a6a4f17129a274baee9#diff-fdf3def0b87eae19cc9989a25a088763bbd46881e814a93f7a6f6a813ca2050a)

But im not sure if its needed here. 

I have seen examples with and without this approach. In my case, this catch block helped me catch configuration (HTTPS, certificates) errors in docker, since without it did not write errors that were thrown (before injecting error middleware?) I checked, it is weird :\